modules/upgrade/partials/k-operator-compatibility-matrix.adoc

```diff
@@ -0,0 +1,14 @@
+|===
+| Redpanda Operator Helm Chart | Redpanda Operator|CRD|Support Redpanda Helm chart
```

Comment:
@JakeSCahill 
Shouldn't the Operator version be in the first column?

WDYT @alejandroEsc ?